pkg/config/config.go

```diff
@@ -936,3 +936,13 @@ func (c *Config) Write() error {
 	}
 	return nil
 }
+
+// Reload reloads the configuration from containers.conf files
+func Reload() (*Config, error) {
+	var err error
+	config, err = NewConfig("")
```

Comment:
Note that this is not thread safe. Looking at the implementation of `Default()`, I suggest to:
```golang
configOnce = sync.Once{}
return Default()
```

Reply:
@QiWang19 PTAL

Reply:
Actually, this would then be racy with `Default` so we need an additional `sync.Mutex` to serialize concurrent invocations of `Reload` and `Default`.